MIP39/mip39.md

```diff
@@ -98,7 +98,11 @@ MIP39c2 subproposals must use the template located at [MIP39c2-Subproposal-Templ
 
 ### MIP39c3: Removing Core Units (Subproposal Process)
 
-This subproposal process allows Governance to remove a Core Unit from the DAO Primitives State (MIP38). A Core Unit can only be removed if it has no active budget implementations (see MIP40) and no active facilitators (see MIP41).
+This subproposal process allows Governance to remove a Core Unit from the DAO Primitives State (MIP38). A Core Unit can only be removed if it has no active budget implementations (see MIP40) and no active facilitators (see MIP41). However, a Core Unit can be offboarded in the same governance cycle as removing its budget and facilitator(s). More precisely, if a Core Unit has no active budget implementation, no active facilitators, no facilitator onboarding proposals in formal submission, and no budget requests in formal submission, no further action is required for the Core Unit to be officially offboarded.
```

Comment:
I don't think this is very clear. So:

1) A Core Unit can only be removed if it has no active budget implementations and no active Facilitators.
2) Can be offboarded in one swift motion by (I presume) submitting a MIP39c3, a MIP40c4, and a MIP41c5 that will be all wrapped up in one single Ratification Poll (not unlike Core Unit onboardings are carried out)
3) Is "More precisely (...) to be officially offboarded" stating the same thing as 1? Probably not? So:

* RE: 1, MKT-001's been de facto/semi-de jure offboarded after the approval of its budget going to zero and its Facilitator being offboarded, with no MIP39c3 being posted after that, as it was abundantly clear that it was the case of a full Core Unit offboarding. Are we "reverting" to the original intended use of MIP39c3 SPs being submitted after MIP41c5 and MIP40c4 SPs?
* RE: 2, it kinda contradicts 1's "only be removed if" as this implies a sequence when 2 is simultaneous and immediate. Also, there's a list of conditions for MIP40c4 SPs. I might be wrong, but it seems possible to have a case of an offboarding "MIP Set" (MIP39c3, MIP40c4, MIP41c5) with its MIP40c4 not fulfilling the new conditions for its setting the budget to zero until some time after the publication of the proposal (e.g., with no Facilitator endorsing the proposal at the moment of the submission). Is this intended?

Reply:
This clause is trying to capture all the possible states of core unit staying in the DAO, but leaving room for the obvious case when it's not running. Probably why it's not clear. I'll have to come back to this.